luigi/rpc.py

```diff
@@ -42,32 +62,61 @@ def __init__(self, message, sub_exception=None):
         self.sub_exception = sub_exception
 
 
+class FetcherException(Exception):
+    def __init__(self, original_exc):
+        self.original_exc = original_exc
+
+
+class URLLibFetcher(object):
+    def fetch(self, full_url, body, timeout):
+        try:
+            body = urlencode(body).encode('utf-8')
+            return urlopen(full_url, body, timeout).read().decode('utf-8')
```

Comment:
Awesome that you fixed this! But you checked somehow that this code path is tested before fixing it right? :)